src/initializers/diamorphosis.ts

```diff
@@ -32,9 +32,36 @@ export default (config, orkaOptions: Partial<OrkaOptions>) => {
     headersRegex: '^X-.*',
     ...config.riviere
   };
+  if (config.kafka) {
+    config.kafka.producer = {
+      brokers: [],
+      certificates: {
+        key: '',
+        cert: '',
+        ca: '',
+        ...config.kafka.producer?.certificates
+      },
+      sasl: {
+        username: '',
+        password: '',
+        ...config.kafka.producer?.sasl
+      },
+      ...config.kafka.producer
+    };
+  }
   diamorphosis(orkaOptions.diamorphosis);
   config.app.env = config.app.env || config.nodeEnv;
 
+  // Override kafka producer config with defaults if brokers is not set
+  if (config.kafka && (!config.kafka.producer?.brokers || config.kafka.producer.brokers.length === 0)) {
```

Comment:
```suggestion
  if (config.kafka && config.kafka.producer?.brokers?.length === 0) {
```
or you could use isEmpty

Reply:
this is not needed since it is checked here `!config.kafka.producer?.brokers`

Reply:
If I change like you suggested if `brokers` is `undefined` it won't enter the `if` condition

Reply:
Yeap you are right. Ignore it.